### DISCUSSION TITLE
vercel部署问题

### DISCUSSION THREAD

**Discussion by tannotour** [Binaryify/NeteaseCloudMusicApi#1323] [Q&A]
在vercel部署后，可以看到项目主页，但是相关接口无法访问

**Comment by bh3Anflower** [comment]
NeteaseCloudMusicApi 也要部署上去，然后将你的请求根路径改成，API 地址。
![image](https://user-images.githubusercontent.com/66837236/153757356-7cc483e3-143b-4e5b-82f5-59e66e69a7bf.png)
![image](https://user-images.githubusercontent.com/66837236/153757397-f8c2abac-71d6-42f2-a04e-720d25fe8eb7.png)
复制的路径等于下面的 3000端口
![image](https://user-images.githubusercontent.com/66837236/153757524-f4ac1ccf-8531-4bcc-ac44-4ba08e942311.png)
axios 参考
![image](https://user-images.githubusercontent.com/66837236/153757658-28e8618d-924b-442d-b118-1a75f7c832e3.png)
